src/Client.js

```diff
@@ -85,68 +86,64 @@ export default class Client {
 	 * Delete an entire published library
 	 * @param  {String} $0.name Name of the library to delete
 	 * @param  {String} $0.force Key to force deleting a public library
-	 * @return {Promise}
+	 * @returns {Promise} A promise
 	 */
-	deleteLibrary({ name, version, force }) {
```

Comment:
This makes me, hmmm

Undocumented argument, gets removed, doesn't cause any other changes in the diff unless I missed them, so maybe not a tested method. Just want to make sure this is alright?

Reply:
it's not being used so i think it's safe :+1: